k8s/devinfra/equinix_k3s/README.md

```diff
@@ -0,0 +1,29 @@
+- Install k3sup (this installs k3sup to `~/bin` if that's not in your path choose somewhere that is)
+```
+curl -sLS https://get.k3sup.dev | sh
+install k3sup ~/bin
+```
+- Make sure your ssh key has access to all of the machines you are going to use.
+- Provision an EIP. Make sure the EIP is tagged with a unique identifier.
+- Run
+```
+EQUINIX_API_KEY=<api_key> \
+EQUINIX_PROJECT_ID=<project_id> \
+EQUINIX_FACILITY=<equinix facility of machines, eg. la4> \
```

Comment:
let's switch to metro since facilities are "deprecated"
see https://feedback.equinixmetal.com/changelog/bye-facilities-hello-again-metros